modules/virt-enabling-volume-snapshot-boot-source.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assembly:
+//
+// * virt/storage/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-enabling-volume-snapshot-boot-source_{context}"]
+= Enabling volume snapshot boot sources
+
+Enable volume snapshot boot sources by setting the parameter in the `DataImportCron` namespace. Although `DataImportCron` was originally designed to maintain only PVC sources, volume snapshot sources scale better than PVC sources for certain storage types.
+
+[NOTE]
+====
+Use volume snapshots on a storage profile that is proven to scale better when cloning from a single snapshot.
+====
+
+.Prerequisites
+
+* You must have access to a volume snapshot with the operating system image.
+* The storage profile must support snapshotting.
```

Comment:
The storage driver must support CSI snapshots.

Reply:
@aglitke @jpeimer I updated this to `The storage must support snapshotting`, is this enough or should I include "CSI"?.

Reply:
This should be enough, because k8s doc says `VolumeSnapshot support is only available for CSI drivers.`
https://kubernetes.io/docs/concepts/storage/volume-snapshots/